src/Resources/Locales/en.json

```diff
@@ -938,6 +938,9 @@
         "noContainerUrlErrorMessage": "Your container url is not set. Click below to set it.",
         "noADTInstanceUrlTitle": "ADT instance not set",
         "noADTInstanceUrlMessage": "Your ADT instance url is not set. Click below to set it.",
-        "configureEnvironment": "Configure environment"
+        "configureEnvironment": "Configure environment",
+        "3dAssetLoadingNetworkErrorTitle": "Error loading model",
+        "3dAssetLoadingNetworkErrorMessage": "This could be a CORS issue, invalid blob url or a network error. Please contact to the owner of the container where this asset file is stored or try again later.",
+        "3dAssetLoadingCommonErrorMessage": "Error loading model. Try Ctrl-F5."
```

Comment:
I don't think we should say "Ctrl-F5". Can we just say "Error loading model. Try clearing your cache and try again"